File: DDS_Restaurant_Solution/DDS_Restaurant_Solution/Models/Menu.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace DDS_Restaurant_Solution.Models
{
    public class Menu
    {
        [Key]
        public int idMenu { get; set; }

        [MaxLength(25)]
        public string nombre { get; set; }

        [Column(TypeName = "Date")]
        public DateTime fechaCreacion { get; set; }
    }
}

File: DDS_Restaurant_Solution/DDS_Restaurant_VB/Models/Menu.vb
Imports System.ComponentModel.DataAnnotations.Schema
Imports System.ComponentModel.DataAnnotations
Public Class Menu
    <Key>
    Public Property idMenu As Integer
    <MaxLength(25)>
    Public Property nombre As String
    <Column(TypeName:="Date")>
    Public Property fechaCreacion As DateTime
End Class
